src/main/resources/assets/js/home/xptour.ts

```diff
@@ -80,6 +80,12 @@ function initDialog(config: GlobalConfig) {
     });
     tourDialog.addClass('xp-tour-dialog');
 
+    tourDialog.onKeyDown((event: KeyboardEvent) => {
+        if(event.key === 'Escape'){
```

Comment:
Please use [helper method](https://github.com/enonic/lib-admin-ui/blob/master/src/main/resources/assets/admin/common/js/ui/KeyHelper.ts#L101) from lib-admin-ui's `KeyHelper` class to check for Escape button. 

If you didn't use `KeyHelper` in your previous PRs, then those changes have to be fixed as well.